src/main/java/org/codeforamerica/messaging/models/TemplateVariant.java

```diff
@@ -36,10 +37,10 @@ public class TemplateVariant {
     @Builder.Default
     String treatment = DEFAULT_TREATMENT;
     String subject;
-    @NotBlank
-    String body;
+    String emailBody;
+    String smsBody;
     @ManyToOne
-    @NotNull
+//    @NotNull
```

Comment:
Why?

Reply:
By adding a validation for the request input for `templateVariants` when a template is being created or modified, an validation is thrown since the templateVariants are provided without a template in the request body. I haven't found a way to only apply the `MessageContentsValidator` without also applying this validation, but I am still exploring. I left this in but commented out because I don't see removing @NotNull as the ideal solution.

Another solution could be to create (another) TemplateVariant-based object that doesn't have a template, but I would really prefer not to go that route. Or not validating at all.

Reply:
Nevermind, I found a solution!